crates/sui-core/src/authority/authority_store.rs

```diff
@@ -262,7 +276,7 @@ impl AuthorityStore {
         Ok(store)
     }
 
-    pub fn get_root_state_hash(&self, epoch: EpochId) -> SuiResult<ECMHLiveObjectSetDigest> {
+    pub fn _get_root_state_hash(&self, epoch: EpochId) -> SuiResult<ECMHLiveObjectSetDigest> {
```

Comment:
Why underscore?

Reply:
sorry, was testing whether I could remove it, forgot to come back to it